src/show/pages/public/CreateProject/Subviews/index.js

```diff
@@ -0,0 +1,443 @@
+            <ReactDatePicker
+              required
+              className='flex justify-center text-center w-full '
+              selected={project.deadline}
+              onChange={deadline =>
+                setProject({ ...project, deadline: deadline })
+              }
+              dateFormat='dd/MM/yyyy'
+              minDate={new Date()}
+              withPortal
+            />
+          </div>
+        </div>
+      </div>
+      <div className='flex justify-between cursor-pointe mt-52'>
+        <div
+          onClick={() => previousStep()}
+          className='flex items-center hover:scale-110 transition-all duration-300'>
+          <BsArrowBarLeft size='2rem' />
+          Back
+        </div>
+        <div
+          onClick={() => nextStep()}
+          className='flex items-center cursor-pointer hover:scale-110 transition-all duration-300'>
+          Next
+          <BsArrowBarRight size='2rem' />
+        </div>
+      </div>
+      <div className='mt-16 px-60'>
+        <ProgressBar percentage={60} color='bg-primary-600' />
+      </div>
+    </div>
+  );
+};
+
+const StatementStep = ({ nextStep, previousStep, project, setProject }) => {
+  return (
+    <div className='mt-36 px-20'>
+      <div className='text-center text-3xl text-gray-500'>
+        Enter your 6 word statement
+      </div>
+      <div className='h-64'>
+        <TextField
+          value={project.statement}
+          className='rounded-2xl mt-2 items-center text-center'
+          onChange={e => setProject({ ...project, statement: e.target.value })}
+        />
+      </div>
+      <div className='flex mt-20 justify-between cursor-pointer'>
+        <div
+          onClick={() => previousStep()}
+          className='flex items-center hover:scale-110 transition-all duration-300'>
+          <BsArrowBarLeft size='2rem' />
+          Back
+        </div>
+        <div
+          onClick={() => nextStep()}
+          className='flex items-center cursor-pointer hover:scale-110 transition-all duration-300'>
+          Next
+          <BsArrowBarRight size='2rem' />
+        </div>
+      </div>
+      <div className='mt-20 px-60'>
+        <ProgressBar percentage={70} color='bg-primary-600' />
+      </div>
+    </div>
+  );
+};
+
+const ImagesStep = ({ nextStep, previousStep, project, setProject }) => {
+  return (
+    <div className='mt-12 px-20'>
+      <div className='text-center text-3xl text-gray-500'>
+        Enter your project photos URL
+      </div>
+      <div className='h-full mt-6'>
+        {project.images.map((image, index) => (
+          <TextField
+            key={index}
+            value={image}
+            className='rounded-2xl mt-2 items-center text-center'
+            onChange={e => {
+              let newImages = project.images;
+              newImages[index] = e.target.value;
+              setProject({ ...project, images: newImages });
+            }}
```

Comment:
Move this logic into a separate function.

Reply:
done